engine/org.eclipse.birt.report.engine.emitter.config/META-INF/MANIFEST.MF

```diff
@@ -3,8 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: BIRT Emitter Configuration Plug-in
 Bundle-SymbolicName: org.eclipse.birt.report.engine.emitter.config;singleton:=true
 Bundle-Version: 4.8.0.qualifier
-Bundle-ClassPath: library.jar,
```

Comment:
I do find library.jar in this plugin, so cannot simply remove this from classpath.